cupy/random/_generator_api.pyx

```diff
@@ -427,6 +430,53 @@ class Generator:
                      (ngood_ptr, nbad_ptr, nsample_ptr))
         return y
 
+    def logseries(self, p, size=None):
+        """Log series distribution.
+
+        Returns an array of samples drawn from the log series distribution.
+            Its probability mass function is defined as
```

Comment:
Seems that the first sentence is unexpectedly made bold. 

<img width="538" alt="cap2021-08-10 15 00 28" src="https://user-images.githubusercontent.com/1224375/128816020-32ade952-636b-4db5-a662-9adf99336016.png">

Would you try to remove the indent as:

```suggestion
        Its probability mass function is defined as
```